cypress/e2e/1-getting-started/aboutPageSpec.cy.js

```diff
@@ -0,0 +1,49 @@
+import { click } from "@testing-library/user-event/dist/click";
+
+describe('About Page spec', () => {
+  beforeEach(() => {
+    // cy.intercept("craft-circle-be.herokuapp.com/graphql", {fixture: "userProfile.json"})
```

Comment:
Great work overall! More updates to testing as we continue to build out functionality.